recipe/meta.yaml

```diff
@@ -13,14 +13,26 @@ source:
 
 build:
   number: 0
-  script: pip install --no-deps .
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Sorry, I thought this was worked out.  Please open an issue on the conda-build tracker.